frontend/src/components/inDepthDetailView/InDepthDetailView.js

```diff
@@ -42,14 +45,13 @@ const useStyles = makeStyles((theme) => ({
   },
 }));
 
-export default function InDepthDetailView(
-  type,
-  title,
-  street,
-  primaryPictureUrl
-) {
+export default function InDepthDetailView() {
   const classes = useStyles();
-  return (
+  const { id } = useParams();
+  const { places } = useContext(PlacesContext);
+  const placeData = places.find((place) => place.id === id);
+
+  return !placeData ? null : (
```

Comment:
works fine, but I'd use the "logical && Operator" here..
`{placeData  && ...`

[https://reactjs.org/docs/conditional-rendering.html](https://reactjs.org/docs/conditional-rendering.html)